### PR TITLE
Feature/https syndication

### PR DESCRIPTION
**Pull request by markysoft**
Change syndication URL from HTTP to HTTPS

### CODE REVIEW

**Comment by neilbmclaughlin** [comment]
Should we be writing our code in a way which honours 301 responses.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 0.1.2 / TBC
```
Comment (st3v3nhunt) [inline CHANGELOG.md]: Is it worth updating the date to next Tuesday given we want to release it then?

**Comment by markysoft** [comment]
Given the traffic involved for an ETL scrape, i.e. tens of thousands of requests, I'd rather hit a URL directly than have a redirect. My preference would be to expose the syndication URL as an ENV var so it can be easily changed, rather than implement redirect functionality.

**Comment by st3v3nhunt** [comment]
Totally agree with:
> Given the traffic involved for an ETL scrape, i.e. tens of thousands of requests, I'd rather hit a URL directly than have a redirect

However, it would have been good to not have it break whilst that change was pending.

None of the suggested possible changes need to be done for this PR but it would be good to agree if they should be done at some point and if so added as an issue.

**Comment by markysoft** [comment]
Agreed, I'd like it to go with the current implementation and the release date set to next Tuesday.
I'll add some issues for the ENV var and the redirect work.

Might be easier to implement for both this and the Pharmacy ETL if they were in a monorepository...

**Pull request merged**